schema.sql

```diff
@@ -58,3 +58,14 @@ CREATE TABLE Author (
     first_name varchar(50) NOT NULL,
     last_name varchar(50) NOT NULL
 );
+
+
+CREATE TABLE Source (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(200) NOT NULL,
+);
+
+CREATE TABLE Movie (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  silet BOOLEAN NOT NULL,
+  FOREIGN KEY(id) REFERENCES Item(id),
```

Comment:
Hi @KaskMIL, 

You have done a great job so far! 💪 👏 All things are functional 👏  you're almost there!

## Top highlights 🎯

- The application is executing according to the requirements ✅ 
- The PR title and description are professional and descriptive ✅ 
- Tests passing ✅ 
- No linter errors ✅ 


## Required Changes ♻️

- Kindly make sure you close your Movie table

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏